framework/mesh/MeshGenerator/OrthogonalMeshGenerator.cc

```diff
@@ -48,6 +48,17 @@ OrthogonalMeshGenerator::OrthogonalMeshGenerator(
     node_sets_.empty(),
     "No nodes have been provided. At least one node set must be provided");
 
+  size_t ns = 0;
+  for (const auto& node_set : node_sets_)
+  {
+    ChiInvalidArgumentIf(
+      node_set.size() < 2,
+      "Node set " + std::to_string(ns) + " only has " +
+        std::to_string(node_set.size()) +
+        " nodes. A minimum of 2 is required to define a cell.");
+    ++ns;
+  }
+
   ChiInvalidArgumentIf(node_sets_.size() > 3,
                        "More than 3 node sets have been provided. Only a "
                        "maximum of 3 are allowed");
```

Comment:
```suggestion
  ChiInvalidArgumentIf(node_sets_.size() > 3,
                       "More than 3 node sets have been provided. The "
                       "maximum allowed is 3.");
```
I would move this before the loop over node sets as well. No reason to go through the nodes if this condition is met.

Reply:
Done